contracts/Staking.sol

```diff
@@ -3,16 +3,15 @@ pragma solidity 0.4.24;
 import "./ERCStaking.sol";
 import "./IStakingLocking.sol";
 import "./ILockManager.sol";
-
 import "./Checkpointing.sol";
 
+import "@aragon/os/contracts/common/Initializable.sol";
 import "@aragon/os/contracts/common/IsContract.sol";
-import "@aragon/os/contracts/common/TimeHelpers.sol";
 import "@aragon/os/contracts/lib/math/SafeMath.sol";
 import "@aragon/os/contracts/lib/token/ERC20.sol";
 
 
-contract Staking is ERCStaking, ERCStakingHistory, IStakingLocking, TimeHelpers, IsContract {
+contract Staking is Initializable, ERCStaking, ERCStakingHistory, IStakingLocking, IsContract {
```

Comment:
It would be better if it inherits from `Autopetrified` so the base contract can never be initialized.